src/components/Homepage.js

```diff
@@ -266,28 +255,29 @@ function Homepage() {
               margin: '24px',
             }}
           >
-            <svg
-              version="1.1"
-              id="Layer_1"
-              xmlns="http://www.w3.org/2000/svg"
-              xmlnsXlink="http://www.w3.org/1999/xlink"
-              x="0px"
-              y="0px"
-              viewBox="0 0 64 64"
-              style={{ enableBackground: 'new 0 0 64 64' }}
-              xmlSpace="preserve"
-            >
-              <style
-                type="text/css"
-                dangerouslySetInnerHTML={{
-                  __html:
-                    '\n                    .st0{fill:none;stroke:#000000;stroke-miterlimit:10;}\n                ',
-                }}
-              />
-              <g>
-                <path
-                  style={{ fill: '#fff' }}
-                  d="M22.2,58c-5.4,0-13.3-1-20-5.8c-2-1.4-2.8-4.1-1.8-6.4c0.9-2.3,3.4-3.6,5.9-3.1l0,0c0,0,1,0.2,2.5,0.1
+            <a href="https://twitter.com/codeclannigeria" target="_blank">
```

Comment:
Codacy found an issue: [Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6048149)